source/pkg/reconciler/streamsource/resources/receive_adapter_test.go

```diff
@@ -35,7 +35,7 @@ func TestMakeReceiveAdapter(t *testing.T) {
 		},
```

Comment:
this file does not seem related to the PR.

Reply:
Removed.